microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -384,7 +384,7 @@ Related matches in **Find on page** aren't available when using InPrivate browsi
 
 To stop Microsoft Edge from finding related matches in **Find on page**:
 
-1. Select **Settings and more** (**...**) > **Settings** > **Privacy, search, and services**, and then scroll down to the **Services** section.  Or, go to `edge://settings/privacy`, and then scroll down to the **Services** section.
+1. Select **Settings and more** (**...**) > **Settings** > **Search and services**, or, go to `edge://settings/searchServices`.
```

Comment:
```suggestion
1. Select **Settings and more** (**...**) > **Settings** > **Search and services**.  Or, go to `edge://settings/searchServices`.
```
two separate sentences enables the first sentence/step/action to be short & simple.  
i want to move entire article toward the GUI approach, and away from the "go to special url" approach.  The GUI approach is self-contained and sufficient, should keep separate from the esoteric "go to special url" approach.

Reply:
general design conventions in article: I doubt that the article should ever state the special url, such as "Go to edge://settings/searchServices".  Seems like noise & nonstandard instructions, detracting from the recommended, GUI nav approach.  Can we just delete:
 (Or, go to `edge://settings/searchServices`.)
@apryzgoda - not sure why the article ever gives these esoteric URLs.  GUI nav approach is standard and sufficient.

Reply:
```
To stop Microsoft Edge from finding related matches in **Find on page**, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Search and services**, and then turn off the toggle **Include related matches in Find on page**.
```